packages/frontend-shared/src/gql-components/modals/SelectCloudProjectModal.vue

```diff
@@ -348,6 +348,8 @@ watch(projectOptions, (newVal, oldVal) => {
   } else {
     pickedProject.value = projectOptions.value.find((p) => p.name === projectName.value)
   }
+
+  newProject.value = projectOptions.value.length === 0
```

Comment:
There are a couple of other places that we could probably do this, but I figure any time `projectOptions` changes, we want to make sure that `newProject` has the correct value

Reply:
Super nitpicky, but we should probably be using `newVal` within this callback rather than referencing the prop value. That's probably what's making the `immedate` param on this `watch` necessary.

Reply:
yeah that's a good point. I think we have `immediate` here as a way to set the initial value of `pickedProject`. My understanding is that we don't set a default value for the ref, so we need this to run immediately to set that value if there is an existing project